docs/file-formats/json-schemas/colors.json

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
I added a JSON schema which is really useful when looking at a colors file with VS Code. It shows hints on hover and validate the JSON against the schema automatically.

I think it also wouldn't be hard to generate the markdown spec from it (the other way around is probably harder)